packages/design-system/src/components/VerticalNav/VerticalNav.tsx

```diff
@@ -59,7 +59,7 @@ export const VerticalNav = (props: VerticalNavProps): React.ReactElement => {
   const navProps = props.ariaNavLabel ? { 'aria-label': props.ariaNavLabel } : {};
 
   const navList = (
-    <ul className={classes} id={props.id}>
+    <ul role="list" className={classes} id={props.id}>
```

Comment:
Spotted this a11y issue while cleaning up. Adding this attr helped remove some unnecessary CSS along the way.